### PR TITLE
Add css for create component and create profile component. 

### PR DESCRIPTION
**Pull request by Nasita-Haque**

### CODE REVIEW

File: frontend/actions/profile-actions.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export const GET_PROFILE_DATA = 'GET_PROFILE_DATA';
```
Comment (gilruben) [inline frontend/actions/profile-actions.js]: delete this file

File: frontend/components/create-page/CreatePage.jsx [hunk 2]
```diff
@@ -30,61 +40,137 @@ const CreatePage = React.createClass({
...
+          contentLabel="Minimal Modal Example"
+          className="modal"
+        >
+          <div className="form-container">
+            <form>
+              <h3>Title</h3>
+              <input className="inputBox" type="text" placeholder="Enter a title" onChange={this.addUploadState.bind(this, 'title')} />
+              <h3>Privacy Setting</h3>
+              <select className="selectButton" onChange={this.addUploadState.bind(this, 'privacy')}>
+                {privacyLevels.map((val, idx) => {
```
Comment (gilruben) [inline frontend/components/create-page/CreatePage.jsx]: press return after open curly bracket, then align appropriately

File: frontend/components/create-page/CreatePage.jsx [hunk 3]
```diff
@@ -30,61 +40,137 @@ const CreatePage = React.createClass({
...
+                    >
+                      {val.value}
+                    </option>
+                  );
+                },
+              )}
+              </select>
+              <h3>Category</h3>
+              <select className="selectButton" onChange={this.addUploadState.bind(this, 'category')}>
+                {categories.map((val, idx) => {
```
Comment (gilruben) [inline frontend/components/create-page/CreatePage.jsx]: press return after open curly bracket, then align appropriately

File: frontend/components/create-page/CreatePage.jsx [hunk 4]
```diff
@@ -30,61 +40,137 @@ const CreatePage = React.createClass({
...
+            </form>
+            <div className="formButton-container">
+              <button className="form-button" onClick={this.onClick}>Upload</button>
+              <button className="form-button" onClick={this.handleCloseModal}>Cancel</button>
+            </div>
+          </div>
+        </ReactModal>
+        <div className="editor-container">
+          <div className="instyleButtons-container">
+            {inlineStyles.map((val, idx) => {
```
Comment (gilruben) [inline frontend/components/create-page/CreatePage.jsx]: press return after open curly bracket, then align appropriately

File: frontend/components/create-page/CreatePage.jsx [hunk 5]
```diff
@@ -30,61 +40,137 @@ const CreatePage = React.createClass({
...
+              return (
+                <button
+                  key={idx}
+                  id="instyleButton"
+                  className={`indiv-instyle-button-${val.name}`}
+                  onClick={this.onInlineStyleClick.bind(this, val.style)}
+                >
+                  { val.name }
```
Comment (gilruben) [inline frontend/components/create-page/CreatePage.jsx]: remove spaces inside of curly brackets

File: frontend/components/create-page/CreatePage.jsx [hunk 6]
```diff
@@ -30,61 +40,137 @@ const CreatePage = React.createClass({
...
+          )}
+            {blockTypes.map((val, idx) => {
+              return (
+                <button
+                  key={idx} 
+                  id="instyleButton"
+                  className={`indiv-blocktype-button-${val.name}`}
+                  onClick={this.onBlockTypeClick.bind(this, val.style)}
+                >
+                  { val.name }
```
Comment (gilruben) [inline frontend/components/create-page/CreatePage.jsx]: remove spaces inside of curly brackets

File: frontend/reducers/profile-reducer.js [hunk 7]
```diff
@@ -0,0 +1,14 @@
+import { GET_PROFILE_DATA } from '../actions/profile-actions';
```
Comment (gilruben) [inline frontend/reducers/profile-reducer.js]: delete this file

File: frontend/styles/create.css [hunk 8]
```diff
@@ -1,6 +1,172 @@
...
+  border-radius: 10px;
+  background-color: #fd2e30;
+  box-shadow: 4px 4px 5px #cecece;
+  color: white;
+  border: 0; 
+}
+
+.modal{
+  position: absolute;
+  /*background-color: rgba(253, 70, 72, 0.15);*/
```
Comment (gilruben) [inline frontend/styles/create.css]: remove this comment

**Review by gilruben — CHANGES_REQUESTED** [review]

**Review by Ceejaymar — APPROVED** [review]

**Review by Ceejaymar — APPROVED** [review]

**Review by gilruben — APPROVED** [review]

**Pull request merged**